litmus-portal/backend/graphql-server/pkg/cluster/util.go

```diff
@@ -21,3 +30,38 @@ func VerifyCluster(identity model.ClusterIdentity) (*model.Cluster, error) {
 	}
 	return &cluster[0], nil
 }
+
+func getIP() {
+
+    var kubeconfig *string
+    nodeIP := []corev1.NodeAddress{}
+
+    if home := homedir.HomeDir(); home != "" {
+        kubeconfig = flag.String("kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) absolute path to the kubeconfig file")
+    } else {
+        kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
+    }
+    flag.Parse()
+    // uses the current context in kubeconfig
+    config, err := clientcmd.BuildConfigFromFlags("", *kubeconfig)
+    if err != nil {
+        panic(err.Error())
+    }
+    // creates the clientset
+    clientset, err := kubernetes.NewForConfig(config)
+    if err != nil {
+        panic(err.Error())
+    }
+
+    nodes, err := clientset.CoreV1().Nodes().List(metav1.ListOptions{})
+    if err != nil {
+        panic(err)
+    }
+    
+    for i := 0; i < len(nodes.Items); i++ {
+        nodeIP = nodes.Items[i].Status.Addresses
+        fmt.Println(nodeIP[0].Address)
```

Comment:
Check which node pod is running, and get the IP of that node. Not possible with out-cluster config.

Reply:
I improved that one now Interbal IP is getting stored

Reply:
I think you have not pushed the updated code @oumkale

Reply:
Yes I will tha